server/supernova/server/buffer_manager.cpp

```diff
@@ -25,7 +25,7 @@
 
 namespace nova {
 
-void buffer_wrapper::allocate(size_t frames, uint channels) {
+void buffer_wrapper::allocate(size_t frames, uint channels, double samplerate) {
```

Comment:
`sampleRate` is not actually used in this function?

Reply:
Good catch!